http/day.http

```diff
@@ -17,4 +17,7 @@ GET {{host}}/days
 GET {{host}}/days/{{dayId}}
 
 ### delete day by id
-DELETE {{host}}/days/{{dayId}}
\ No newline at end of file
+DELETE {{host}}/days/{{dayId}}
+
+### find day by date
+GET {{host}}/days/date?{{date}}
```

Comment:
It is work? Please, check it one more time. I suppose it will be  GET {{host}}/days/date?date={{date}}. And I don't see any changes in http-client.env.json. You need to add "date": "2023-02-21" (date changing manual every time)